Internal/Debug App/Sources/Network/Networking.swift

```diff
@@ -262,7 +261,6 @@ class Networking {
         let url = environment.baseUrl.appendingPathComponent("/api/client-session")
 
         var bodyData: Data!
-        var headers: [String: String]?
```

Comment:
The reason we had those here is for when we want to test something in particular and need to pass some extra headers.